xml/System.Reflection.Emit/MethodBuilder.xml

```diff
@@ -1484,8 +1484,9 @@
         <ReturnType>System.Boolean</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
-        <value>To be added.</value>
+        <summary>When overridden in a derived class, gets or sets a Boolean value that specifies whether the local variables in this method are zero initialized. The default value of this property is <see langword="true" />.</summary>
+        <value>
+          <see langword="true" /> if the local variables in this method should be zero initialized; otherwise <see langword="false" />.</value>
```

Comment:
```suggestion
          <see langword="true" /> if the local variables in this method should be zero initialized; otherwise <see langword="false" />. The default is <see langword="true" />.</value>
```